XIVSlothCombo/Combos/CustomComboPreset.cs

```diff
@@ -1481,108 +1481,118 @@ public enum CustomComboPreset
         #region Advanced ST
         [ConflictingCombos(GNB_ST_Simple)]
         [ReplaceSkill(GNB.KeenEdge)]
-        [CustomComboInfo("Advanced Mode - Single Target", "Replace Keen Edge with its combo chain.\n Uses Burst Strike when you have 3 cartridges & your last combo action was Brutal Shell.", GNB.JobID)]
+        [CustomComboInfo("Advanced Mode - Single Target", "Replace Keen Edge with its combo chain.\nThis is the ideal selection for experienced users of the job.", GNB.JobID)]
         GNB_ST_Advanced = 7003,
 
-        #region Lightning Shot
-        [ParentCombo(GNB_ST_Advanced)]
-        [CustomComboInfo("Lightning Shot Uptime Option", "Adds Lightning Shot to the main combo when you are out of range.", GNB.JobID)]
-        GNB_ST_RangedUptime = 7004,
-        #endregion
-
-        #region Gnashing Fang
-        [ParentCombo(GNB_ST_Advanced)]
-        [CustomComboInfo("Continuation Option", "Adds Continuation to the main combo.\n 'Gnashing Fang Starter' option must be enabled or started manually.", GNB.JobID)]
-        GNB_ST_Continuation = 7005,
-
-        [ParentCombo(GNB_ST_Continuation)]
-        [CustomComboInfo("Gnashing Fang Starter Option", "Begins Gnashing Fang for you on the main combo.", GNB.JobID)]
-        GNB_ST_GnashingStarter = 7006,
-        #endregion
-
         #region Cooldowns
         [ParentCombo(GNB_ST_Advanced)]
-        [CustomComboInfo("Cooldowns Option", "Adds various cooldowns to the main combo.", GNB.JobID)]
-        GNB_ST_Advanced_CooldownsGroup = 7007,
+        [CustomComboInfo("Cooldowns Option", "Adds various cooldowns into the rotation.", GNB.JobID)]
+        GNB_ST_Advanced_CooldownsGroup = 7004,
 
         [ConflictingCombos(GNB_NM_Features)]
         [ParentCombo(GNB_ST_Advanced_CooldownsGroup)]
-        [CustomComboInfo("No Mercy Option", "Adds No Mercy to the main combo when at full ammo.", GNB.JobID)]
-        GNB_ST_NoMercy = 7008,
+        [CustomComboInfo("No Mercy Option", "Adds No Mercy into the rotation when appropriate.", GNB.JobID)]
+        GNB_ST_NoMercy = 7005,
 
         [ParentCombo(GNB_ST_Advanced_CooldownsGroup)]
-        [CustomComboInfo("Danger/Blasting Zone Option", "Adds Danger/Blasting Zone to the main combo.", GNB.JobID)]
-        GNB_ST_BlastingZone = 7009,
+        [CustomComboInfo("Sonic Break Option", "Adds Sonic Break into the rotation when appropriate.", GNB.JobID)]
+        GNB_ST_SonicBreak = 7006,
```

Comment:
These (and many other options aside from those highlighted) should be re-ordered with `CustomComboInfo`'s `Order` parameter, instead of changing the meaning of users' currently selected option IDs.
e.g. If I have `Lightning Shot Uptime` enabled now then after this update I would have `Cooldowns Option` enabled without ever having turned it on.

Reply:
Opened [a PR](https://github.com/Akechi-kun/XIVSlothCombo/pull/15) to keep existing IDs, reordering just by location in file.